moninag/moninag/urls.py

```diff
@@ -20,6 +20,7 @@
 urlpatterns = [
     url(r'^', include('home.urls')),
     url(r'^api/1/service/', include('service.urls')),
+    url(r'^api/1/check/', include('check.urls')),
     # Registration & Auth
     url(r'^accounts/', include('registration.urls'))
 ]
```

Comment:
Did you accidentally delete server api url? :slightly_smiling_face: 
`url(r'^api/1/server/', include('server.urls')),`